### DISCUSSION TITLE
There is an issue with registering on the forums on the main site - activation email does not arrive

### DISCUSSION THREAD

**Discussion by guysoft** [audacious-media-player/audacious#110] [General]
There is an issue with registering on the forums on the main site - activation email does not arrive
Thought it might be worth to post here so you know.
I tried checking in spam and pressing the "Send link again" button and no luck.

**Comment by mschwendt** [comment]
That is intentional. See: https://redmine.audacious-media-player.org/boards/1/topics/3051

**Reply by guysoft** [reply]
Thanks for updating. Sorry I missed the pinned post there
